pkg/ansible/controller/status/types.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+import (
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/ansible/runner/eventapi"
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	host = "localhost"
+)
+
+// AnsibleStatus - encapsulation of the ansible result.
+type AnsibleStatus struct {
+	Ok               int                `json:"ok"`
+	Changed          int                `json:"changed"`
+	Skipped          int                `json:"skipped"`
+	Failures         int                `json:"failures"`
+	TimeOfCompletion eventapi.EventTime `json:"completion"`
+}
+
+// NewAnsibleStatusFromStatusJobEvent - creates a Ansible status from job event.
+func NewAnsibleStatusFromStatusJobEvent(je eventapi.StatusJobEvent) *AnsibleStatus {
+	// ok events.
+	o := 0
```

Comment:
What do you think about just creating the `AnsibleStatus` here, letting the values default to 0, and then set them in the code below as appropriate?